### DISCUSSION TITLE
How can I export selected keywords to another computer?

### DISCUSSION THREAD

**Discussion by CopaceticMeatbag** [rsjaffe/MIDI2LR#743] [Q&A]
I've got a list of 64 keywords selected in Lightroom > Plugin Extras > Keywords, and I want to duplicate this to another computer.

I can easily export the list of LR keywords via Metadata > Export, but MIDI2LR has no keywords selected in Plugin Extras > Keywords, meaning I have to manually go through selecting the correct keyword for each spot again. Is there an easier way to do this?

Thanks all!

**Comment by antzpantz** [comment]
Have you tried Importing/Exporting your MIDI2LR configuration?

[image: image.png]

**Reply by CopaceticMeatbag** [reply]
Yep I have - I can load the MIDI2LR xml and get the same setup re midi button mappings, but the LR Plugin Extras > Keywords selections don't update with the xml. 

I looked through the xml to be sure and there's no keywords stored in there, it's definitely just mapping a button to "keyword1" etc without an actual value. I suspect it may actually be Lightroom that stores the selections somewhere...for the life of me I can't find where yet though 😂
